Dockerfile-alpine

```diff
@@ -93,9 +94,11 @@ RUN curl -L https://download.newrelic.com/php_agent/archive/${NEWRELIC_VERSION}/
     cd newrelic-php5-${NEWRELIC_VERSION}-linux-musl && \
     echo "\n" | ./newrelic-install install && \
     chown root:root /root/newrelic-php5-${NEWRELIC_VERSION}-linux-musl/agent/x64/newrelic-20151012.so && \
-    cp /root/newrelic-php5-${NEWRELIC_VERSION}-linux-musl/agent/x64/newrelic-20151012.so /usr/lib/php7/modules/newrelic.so && \
+    mv /root/newrelic-php5-${NEWRELIC_VERSION}-linux-musl/agent/x64/newrelic-20151012.so /usr/lib/php7/modules/newrelic.so && \
```

Comment:
previously a copy, leaving this and older php agents in place. move the important one, delete the others.

~10MB shaved